cernopendata/modules/fixtures/data/docs/cod-privacy-policy/cod-privacy-policy.md

```diff
@@ -1 +1,61 @@
-CERN does not track, collect or retain personal information from users of CERN Open Data Portal, except as otherwise provided herein. In order to enhance CERN Open Data Portal and monitor traffic, information such as IP addresses and cookies may be tracked and retained, as well as log files shared in aggregation with other community services. User provided information, like corrections of metadata or paper claims, will be integrated into the database without displaying its source and may shared with other services. CERN Open Data Portal will take all reasonable measures to protect the privacy of its users and to resist service interruptions, intentional attacks, or other events that may compromise the security of the CERN Open Data Portal website. If you have any questions about the CERN Open Data Portal privacy policy, please contact [opendata-support@cern.ch](mailto:opendata-support@cern.ch)
+CERN does not track, collect or retain personal information from users of CERN Open Data Portal, except as otherwise provided herein. In order to enhance CERN Open Data Portal and monitor traffic, information such as IP addresses and cookies may be tracked and retained, as well as log files shared in aggregation with other community services. User provided information, like corrections of metadata or paper claims, will be integrated into the database without displaying its source and may shared with other services.
```

Comment:
I feel we should introduce some structure, e.g. clearly label the first part and the second part.  For example:

```
- Privacy policy

(old text comes here)

- CERN Privacy Notice PN00913

(new text comes here)
```

This will help to present to users a condensed summary (on top), and a sort of fine read (new text) that says "Each service at CERN...".  They would know better why.

After you amend, you can better scale the sub-headers as well.

Reply:
Also, can you please rebase against `qa` so that we can merge to production faster?